srcpkgs/pygtk/template

```diff
@@ -2,20 +2,24 @@
 pkgname=pygtk
 version=2.24.0
 revision=13
-lib32disabled=yes
 build_style=gnu-configure
 configure_args="PYTHON=python2"
+# This is required to avoid compiling all of gtk-2.0
+# Ignore xlint on this one
+pycompile_module="gtk-2.0/gtk"
+pycompile_dirs="usr/lib/pygtk"
```

Comment:
usr/lib/pygtk is not part of main package, so no need to set here

Reply:
Done